variables.tf

```
@@ -14,10 +14,47 @@
  * limitations under the License.
  */
 
-variable "project_id" {
-  description = "The project ID to deploy to"
+variable "parent_id" {
+  description = "Id of the resource under which the folder will be placed."
 }
 
-variable "bucket_name" {
-  description = "The name of the bucket to create"
+variable "parent_type" {
+  description = "Type of the parent reosurce, defaults to organization."
+  default     = "organization"
+}
+
+variable "names" {
+  description = "Folder names."
+  default     = []
+}
+
+variable "set_roles" {
+  description = "Set roles to actors passed in role_members variable."
+  default     = false
+}
+
+variable "per_folder_admins" {
+  description = "List of comma delimited IAM-style members per folder who will get extended permissions."
```

Lists are always comma delimited.

```suggestion
  description = "List of IAM-style members per folder who will get extended permissions."
```